dist/README.SETUP

```diff
@@ -1,45 +1,49 @@
 # Manual Setup Guide
```

Comment:
* README.SETUP.md for nice view on github
* link README.SETUP for "backwards compatiblity"